<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADDialog</class>
 <widget class="QDialog" name="ADDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ADDialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>341</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;Century Gothic&quot;;</string>
   </property>
   <property name="title">
    <string>Select </string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Britannic Bold&quot;;</string>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Britannic Bold&quot;;</string>
    </property>
    <property name="text">
     <string>Paticipator</string>
    </property>
   </widget>
   <widget class="QComboBox" name="TitleComboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Batang&quot;;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="paticipatorComboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;Batang&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="comfirmButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>212</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Bell MT&quot;;</string>
    </property>
    <property name="text">
     <string>Comfirm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnButtom">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Bell MT&quot;;</string>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
